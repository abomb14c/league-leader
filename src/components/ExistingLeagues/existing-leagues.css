.existing-leagues-container {
  width: 100%;
  display: flex;
  flex-wrap: wrap;
  margin-bottom: 50px;
}