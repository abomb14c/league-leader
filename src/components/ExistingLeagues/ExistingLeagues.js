import React from 'react';
import LeagueCard from '../LeagueCard/LeagueCard';
import './existing-leagues.css';

const ExistingLeagues = ({leagues}) => {
  const displayLeagues = leagues.map((league, index) => {
    return <LeagueCard {...league} key={index}/>;
  });
  return (
    <div className="existing-leagues-container">
      {displayLeagues}
    </div>
  );
};

export default ExistingLeagues;