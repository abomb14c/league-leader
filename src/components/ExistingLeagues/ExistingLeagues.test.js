import React from 'react';
import { shallow } from 'enzyme';
import  ExistingLeagues  from './ExistingLeagues';

describe('ExistingLeagues', () => {
  let wrapper;
  let mockProps; 

  beforeEach(() => {
    mockProps = {
      leagues: [{}, {}, {}]
    };

    wrapper = shallow(<ExistingLeagues
      {...mockProps}
    />);
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});