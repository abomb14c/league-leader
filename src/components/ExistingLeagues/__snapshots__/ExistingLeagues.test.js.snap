// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExistingLeagues should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ExistingLeagues
    leagues={
        Array [
            Object {},
            Object {},
            Object {},
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <LeagueCard />,
        <LeagueCard />,
        <LeagueCard />,
      ],
      "className": "existing-leagues-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "2",
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <LeagueCard />,
          <LeagueCard />,
          <LeagueCard />,
        ],
        "className": "existing-leagues-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "2",
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
